scripts/wizard.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+set -e
+########################################
+#             Please Read!             #
+# This script will not run standalone, #
+# please use launch.sh.                #
+########################################
+
+#########################################################
+# helper functions                                      #
+#########################################################
+
+print_user_input_query() {
+    local question="$1"
+    local default_answer="$2"
+    local msg="\033[1;36m${question}\033[0m"
+    [[ ! -z "$default_answer" ]] && msg="${msg} [\033[2;37m${default_answer}\033[0m]"
+    msg="${msg}:"
+    echo -e "${msg}"
+}
+
+print_error() {
+    local msg="$1"
+    echo -e "\033[1;31m${msg}\033[0m"
+}
+
+#########################################################
+# end of helper functions                               #
+#########################################################
+
+# Wizard Code
+AVAILABLE_CONFIGS=("godot" "unreal")
+
+print_green "Welcome to the unifree Wizard!"
+print_green "Please answer a couple of questions so that we can get you started!"
+
+# Ask for OpenAPI Key
+if [[ -z "$OPENAI_API_KEY" ]]; then
+    while [[ $OPENAI_API_KEY_TMP = "" ]]; do
+        print_user_input_query "What is your OpenAPI key?"
+        read OPENAI_API_KEY_TMP
+    done
+    OPENAI_API_KEY="${OPENAI_API_KEY_TMP}"
+else
+    print_user_input_query "What is your OpenAPI key?" "${OPENAI_API_KEY}"
+    read OPENAI_API_KEY_TMP
+    : ${OPENAI_API_KEY_TMP:=$OPENAI_API_KEY}
+fi
+
+# Ask for the source Unity path
+while [[ $ORIGIN_DIR = "" ]]; do
+    print_user_input_query "Full path of the source Unity project"
+    read ORIGIN_DIR
+    if [[ ! -d "${ORIGIN_DIR}" ]]; then
+        print_error "Canno't locate directory ${ORIGIN_DIR}"
+        ORIGIN_DIR=""
+    fi
+done
+
+# Ask for the Destenation path
+while [[ $DEST_DIR = "" ]]; do
+    print_user_input_query "Full path of the target destination path"
+    read DEST_DIR
+done
+
+# Ask for the target engine
+while [[ $CONFIG_NAME = "" ]]; do
+    print_user_input_query "What language would you like to translate it to?"
+    for idx in "${!AVAILABLE_CONFIGS[@]}"; do
+        echo -e "   [${idx}] ${AVAILABLE_CONFIGS[$idx]}"
+    done
+    read CONFIG_NAME
+
+    # Check wether the user actually selected an exists config
+    expr='^[0123456789]+$'
+    if [[ $CONFIG_NAME =~ $expr && -n "${AVAILABLE_CONFIGS[$CONFIG_NAME]}" ]]; then
+        CONFIG_NAME="${AVAILABLE_CONFIGS[$CONFIG_NAME]}"
+    else
+        print_error "Unknwon input ${CONFIG_NAME}."
+        CONFIG_NAME=""
+    fi
+done
+
+print_delimiter
+
+# Print all variables
```

Comment:
In my opinion, we don't need this since it won't mean anything to the user if they are using the wizard. What do you think?

Reply:
sure, ill remove it